docs/reference-architectures/serverless/web-app.md

```diff
@@ -166,29 +155,34 @@ For more information, see [Working with client identities](/azure/azure-function
 
 ### CORS
 
-In this reference architecture, the web application and the API do not share the same origin. That means when the application calls the API, it is a cross-origin request. Browser security prevents a web page from making AJAX requests to another domain. This restriction is called the *same-origin policy* and prevents a malicious site from reading sensitive data from another site. To enable a cross-origin request, add a Cross-Origin Resource Sharing (CORS) [policy][cors-policy] to the API Management gateway:
```

Comment:
The way we setup theurlist.com was to route web traffic to domain.com and api requests to domain.com/api. This removes cross-origin concerns.